src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -771,7 +804,7 @@ private void ReportDiagnostic(ErrorCode errorCode, SyntaxNode syntaxNode, params
             }
         }
 
-        private void InheritNullableStateOfTrackableStruct(TypeSymbol targetType, int targetSlot, int valueSlot, bool isByRefTarget, int slotWatermark)
+        private void InheritNullableStateOfTrackableStruct(TypeSymbol targetType, int targetSlot, int valueSlot, bool isDefaultValue, bool isByRefTarget, int slotWatermark)
```

Comment:
>bool isDefaultValue [](start = 113, length = 19)

If we are propagating this flag separately anyway, why do we need slots for the "fake" state?